buidl/psbt_helper.py

```diff
@@ -90,7 +89,23 @@ def create_p2sh_multisig_psbt(
 
         # pubkey lookups needed for validation
         input_pubkey_hexes = []
-        for xfp_hex, root_path in input_dict["path_dict"].items():
+
+        # If this input's XFP/path information is a dictionary, then
+        # assume we want BIP67 ordering.
+        if isinstance(input_dict["path_dict"], dict):
+            pubkey_iterable = input_dict["path_dict"].items()
+            assume_bip67 = True
+        # If this input's XFP/path information is a list/tuple, then
+        # use the ordering given.
+        elif isinstance(input_dict["path_dict"], (tuple, list)):
```

Comment:
I like the layout of this, but isn't it very confusing that an instance of `path_dict` could be a `tuple` or a `list`?

Does this logic accomplish the same but better?
https://github.com/buidl-bitcoin/buidl-python/commit/fe4605e2ad7faf917b78fe209b7ce1f522cd7f6c

Reply:
Yes, it is confusing. Was going for a minimal change.  I definitely prefer being explicit with `path_list` vs. `path_dict`. I'll rescind this PR -- timeframe getting https://github.com/buidl-bitcoin/buidl-python/commit/fe4605e2ad7faf917b78fe209b7ce1f522cd7f6c merged & released?

Reply:
I put a branch up here, can merge as soon as I add test coverage: https://github.com/buidl-bitcoin/buidl-python/pull/121